Running tests under Python 3.9.18: /home/jmw/anaconda3/envs/vct_tf/bin/python
[ RUN      ] ModelsTest.test_session
[  SKIPPED ] ModelsTest.test_session
[ RUN      ] ModelsTest.test_train_eval
  0%|                                                                                                                                       | 0/1 [00:00<?, ?batch/s]
  0%|                                                                                                                                    | 0/3782 [00:00<?, ?batch/s]
resume /home/jmw/ing/myfi/vfi/ckpt
load a checkpoint at/home/jmw/ing/myfi/vfi/ckptfor/home/jmw/ing/myfi/vfi/ckpt/ckpt-0
Creates a checkpoint at/home/jmw/ing/myfi/vfi/ckptfor0
/home/jmw/ing/myfi/vfi/ckpt/ckpt-0



  0%|                                                                                                                                       | 0/1 [00:13<?, ?batch/s]
Traceback (most recent call last):
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/jmw/ing/myfi/vfi/src/models_test.py", line 109, in <module>
    tf.test.main()
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/tensorflow/python/platform/test.py", line 56, in main
    return _googletest.main(argv)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/tensorflow/python/platform/googletest.py", line 62, in main
    benchmark.benchmarks_main(true_main=main_wrapper, argv=argv)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/tensorflow/python/platform/benchmark.py", line 503, in benchmarks_main
    true_main()
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/tensorflow/python/platform/googletest.py", line 60, in main_wrapper
    return app.run(main=g_main, argv=args)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/tensorflow/python/platform/googletest.py", line 51, in g_main
    absltest_main(argv=argv)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/absl/testing/absltest.py", line 2051, in main
    _run_in_app(run_tests, args, kwargs)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/absl/testing/absltest.py", line 2146, in _run_in_app
    function(argv, args, kwargs)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/absl/testing/absltest.py", line 2549, in run_tests
    result = _run_and_get_tests_result(
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/absl/testing/absltest.py", line 2518, in _run_and_get_tests_result
    test_program = unittest.TestProgram(*args, **kwargs)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/unittest/main.py", line 101, in __init__
    self.runTests()
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/unittest/main.py", line 271, in runTests
    self.result = testRunner.run(self.test)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/absl/testing/_pretty_print_reporter.py", line 82, in run
    return super(TextTestRunner, self).run(test)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/unittest/runner.py", line 184, in run
    test(result)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/unittest/suite.py", line 84, in __call__
    return self.run(*args, **kwds)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/unittest/suite.py", line 122, in run
    test(result)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/unittest/suite.py", line 84, in __call__
    return self.run(*args, **kwds)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/unittest/suite.py", line 122, in run
    test(result)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/unittest/case.py", line 651, in __call__
    return self.run(*args, **kwds)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/unittest/case.py", line 592, in run
    self._callTestMethod(testMethod)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/unittest/case.py", line 550, in _callTestMethod
    method()
  File "/home/jmw/ing/myfi/vfi/src/models_test.py", line 92, in test_train_eval
    self._restore_evaluate(ckpt_p)
  File "/home/jmw/ing/myfi/vfi/src/models_test.py", line 63, in _restore_evaluate
    _, metrics = test_step(img0,gt,img1,0.5)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/tensorflow/python/util/traceback_utils.py", line 141, in error_handler
    return fn(*args, **kwargs)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/tensorflow/python/eager/def_function.py", line 915, in __call__
    result = self._call(*args, **kwds)
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/tensorflow/python/eager/def_function.py", line 947, in _call
    return self._stateless_fn(*args, **kwds)  # pylint: disable=not-callable
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/tensorflow/python/eager/function.py", line 2496, in __call__
    return graph_function._call_flat(
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/tensorflow/python/eager/function.py", line 1862, in _call_flat
    return self._build_call_outputs(self._inference_function.call(
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/tensorflow/python/eager/function.py", line 499, in call
    outputs = execute.execute(
  File "/home/jmw/anaconda3/envs/vct_tf/lib/python3.9/site-packages/tensorflow/python/eager/execute.py", line 54, in quick_execute
    tensors = pywrap_tfe.TFE_Py_Execute(ctx._handle, device_name, op_name,
KeyboardInterrupt